pkg/multitenant/tenantcapabilities/tenantcapabilitiesauthorizer/authorizer.go

```diff
@@ -354,47 +338,41 @@ func (a *Authorizer) HasTSDBQueryCapability(ctx context.Context, tenID roachpb.T
 	case authorizerModeAllowAll:
 		return nil
 	case authorizerModeV222:
```

Comment:
I wonder if we should do a cleanup and remove this setting option in a separate PR